config/formats.js

```diff
@@ -12,9 +12,11 @@ exports.Formats = [
 	},
 	{
 		name: "[Gen 7] Random Battle",
-		desc: ["Randomized teams of level-balanced Pok&eacute;mon with sets that are generated to be competitively viable."],
-
-		mod: 'gen7',
+		desc: ["Randomized teams of level-balanced Pok&eacute;mon with sets that are generated to be competitively viable."
+			"&bullet; <a href=\"http://www.smogon.com/forums/threads/3591157/\">Sets and Suggestions"</a>",
```

Comment:
If you can't tell by the dark red here (and Travis CI yelling at you), you did this wrong.

You have a stray `"` that shouldn't be here.

Also, all `desc` fields should use `` ` `` quotes, so you don't have to escape `\"`.

Reply:
ah rip.